src/main/java/com/crud/library/controller/RentalController.java

```diff
@@ -0,0 +1,34 @@
+package com.crud.library.controller;
+
+import com.crud.library.dto.RentalDto;
+import com.crud.library.service.RentalService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/v1/rental")
+public class RentalController {
+
+    private final RentalService service;
+
+    @PostMapping(value = "/{userId}/{exemplarId}", consumes = APPLICATION_JSON_VALUE)
```

Comment:
tu bym dał @RequestParam bo to nie jest id rentala

Reply:
oooo super uwaga